code/campaignclassic/src/main/java/com/adobe/marketing/mobile/InputBoxPushTemplate.java

```diff
@@ -0,0 +1,54 @@
+/*
+  Copyright 2023 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+package com.adobe.marketing.mobile;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.adobe.marketing.mobile.util.DataReader;
+import java.util.Map;
+
+class InputBoxPushTemplate extends AEPPushTemplate {
```

Comment:
you are correct, this was initially added to for remind later functionality which was then shifted to the basic template. i left it in as it will be used in phase 2.